src/translations/fr/suggest.json

```
@@ -0,0 +1,18 @@
+{
+  "ty-contribution": "Merci",
+  "object-info": "Informations sur l'objet",
+  "prev-pic": "Voir l'objet prit en photo",
+  "mess-info": "En renseignant toutes ses informations vous acceptez de respecter nos conditions générales. Attention toutes bêtises seront sanctionnées.",
+  "getPackError": "Erreur avec le chargement des compositions et matières :",
+  "getBrandError": "Erreur avec le chargement des marques : ",
+  "postPackError": "Erreur lors de la création de la composition ou matière : ",
+  "postBrandError": "Erreur lors de la création de la marque :",
+  "postObjectError": "Erreur lros de l'ajout des informations de l'objet",
+  "name": "Nom",
+  "bin": "Poubelle recommandée",
+  "packaging": "Matière et composition",
+  "brand": "Marque",
+  "messageReward": "En récompense tu gagnes 5 points and 0,10€",
```

```suggestion
  "messageReward": "En récompense tu gagnes 5 points et 0,10€",
```
C'est mieux en français ^^